app/src/main/java/de/danoeh/antennapod/fragment/swipeactions/AddToQueue.java

```diff
@@ -0,0 +1,43 @@
+package de.danoeh.antennapod.fragment.swipeactions;
+
+import android.content.Context;
+
+import androidx.fragment.app.Fragment;
+
+import java.util.Arrays;
+import java.util.List;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.menuhandler.FeedItemMenuHandler;
+import de.danoeh.antennapod.model.feed.FeedItem;
+import de.danoeh.antennapod.model.feed.FeedItemFilter;
+
+public class AddToQueue extends SwipeAction {
```

Comment:
How about `AddToQueueSwipeAction`? Otherwise, just reading the class name does not really tell what the class is used for.

Reply:
Yea they are in the folder swipeactions but I guess I could also rename them